documentation/observability/5-deploy-to-kyma.md

```diff
@@ -4,34 +4,34 @@
 
 To start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the SAP Cloud Logging service, you need to follow these steps:
 
-1. Enable the Kyma [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) by following the steps described as [Quick Install of kyma-project.io](https://kyma-project.io/#/02-get-started/01-quick-install).
+1. Add the Kyma [Telemetry module](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-telemetry-module) by following the steps described in [Add and Delete a Kyma Module Using Kyma Dashboard](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module).
 
-2. An instance of SAP Cloud Logging with OpenTelemetry is enabled to ingest distributed traces.
+2. An instance of SAP Cloud Logging with OpenTelemetry is enabled to ingest distributed traces and metrics.
```

Comment:
This is phrased more like a prerequisite. For a step, I'd suggest using imperative.

Reply:
Oh wait, that comes in line 17? That seems a bit redundant to me.

Reply:
Then I'd clearly separate the overview/intro paragraph from the step instructions.
Basically, saying

> To start streaming logs, metrics and traces from SAP BTP, Kyma runtime to the SAP Cloud Logging service, you need the Kyma Telemetry module and an instance of SAP Cloud Logging with OpenTelemetry:

1. Add the Telemetry module. For details, see [Add and Delete](URL).
2. To create the SAP Cloud Logging service instance, first, log in to the Kyma cluster.
   > [! TIP]
   > We recommend...
3. Create a namespace...

etc.

Reply:
Ignore; this requires bigger changes which should happen in a different PR.

Reply:
done